core/gesture.js

```diff
@@ -665,7 +665,7 @@ Blockly.Gesture.prototype.handleWsStart = function(e, ws) {
  * @private
  */
 Blockly.Gesture.prototype.fireWorkspaceClick_ = function(ws) {
-  var clickEvent = new Blockly.Events.Click(null, ws.id);
+  var clickEvent = new Blockly.Events.Click(null, ws.id, 'workspace');
```

Comment:
Could we make this a constant somewhere?

Reply:
For `'workspace'`? I'd rather add that in a separate PR.  I would do so for all the strings passed for `targetType` and probably also do a similar thing for `bubbleType` on `BubbleOpen`.